src/metabase/upload.clj

```diff
@@ -127,29 +133,31 @@
 
 (defn value->type
   "The most-specific possible type for a given value. Possibilities are:
-    - ::boolean
-    - ::int
-    - ::float
-    - ::varchar_255
-    - ::text
-    - ::date
-    - ::datetime
-    - nil, in which case other functions are expected to replace it with ::text as the catch-all type
+
+    - `::boolean`
+    - `::int`
+    - `::float`
+    - `::varchar-255`
+    - `::date`
+    - `::datetime`
+    - `::zoned-datetime`
```

Comment:
This should be `offset-datetime`, not `zoned-datetime`. A time zone is Europe/Amsterdam where an offset is like UTC+2. The difference is significant because a time zone has multiple offsets depending on the time of the year

Reply:
good catch, fixed and pushed (along with other comments)